f8a_worker/workers/mercator.py

```diff
@@ -197,6 +197,7 @@ def run_mercator(self, arguments, cache_path,
         if arguments['ecosystem'] == 'go':
             # no Go support in Mercator-go yet, we handle it separately here
             tc = TimedCommand(['gofedlib-cli', '--dependencies-main', '--dependencies-packages',
+                               '--dependencies-test', '-s',
```

Comment:
Could you use `--skip-errors` instead of the short `-s` ?

Reply:
Sure, let me fix that.